lib/matplotlib/contour.py

```diff
@@ -273,6 +272,7 @@ def get_label_width(self, lev, fmt, fsize):
         width *= 72 / fig.dpi
         return width
 
+    @_api.deprecated("3.7", alternative="Artist.set")
```

Comment:
The deprecation is a bit terse

I think

``labeler.set_label_props(label, text, color)``

is equivalent to 

```label.set(text=text, color=color, fontproperties=labeler.labelFontProps, clip_box(labeler.axes)```

Which is not quite straight forward unless you check the source. Can we document this better (even only in the changelog would be ok).

Reply:
sure, done.